code/script.js

```diff
@@ -1,47 +1,120 @@
+const button = document.getElementById('button');
+const nameInput = document.getElementById('name-input');
+const inputWrapper = document.getElementById('input-wrapper');
 
 // Global variables, if you need any, declared here
+let currentQuestion = 1; 
 
-// Functions declared here
+
+// FUNCTIONS
+
+// 4., 6., 8.
+    const nextQuestion = (message) => {
+
+      if(currentQuestion === 1) {
+        nameInput.value = ''
+        setTimeout(() => showFoodOptions(message), 1000)
+
+      } else if(currentQuestion === 2) {
+        showMessage(message, 'user')
+        setTimeout(() => showMessage(`So you want to order ${message}?`, 'bot'), 1000)
+        showPriceSushi() 
+
+      } else if(currentQuestion === 3) {
+        showMessage(message, 'user')
+        ending(message)
+      } 
+      
+
+    }
 
 // This function will add a chat bubble in the correct place based on who the sender is
-const showMessage = (message, sender) => {
-  if (sender === 'user') {
-    chat.innerHTML += `
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
-
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
-
-// Set up your eventlisteners here
-
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+    const showMessage = (message, sender) => {
+      if (sender === 'user') {
+        chat.innerHTML += ` 
+          <section class="user-msg">
+            <div class="bubble user-bubble">
+              <p>${message}</p>
+            </div>
+            <img src="./assets/user.png" alt="User" />  
+          </section>
+        `
+
+      } else if (sender === 'bot') {
+        chat.innerHTML += ` 
+          <section class="bot-msg">
+          <img src="./assets/sushi-roll-icon.jpeg" alt="">
+            <div class="bubble bot-bubble">
+              <p>${message}</p>
+            </div>
+          </section>
+        `
+      chat.scrollTop = chat.scrollHeight
+    }
+ }
+
+//5.
+    const showFoodOptions = () => {
+      currentQuestion++ // current question is 2
```

Comment:
Good with small comments at each interaction with the bot. Makes it easy for me as a code reviewer to read the code.